test/specmap/all-of.js

```diff
@@ -212,6 +212,11 @@ describe('allOf', function () {
           a: 123
         }
       },
+      properties: {
+        allOf: {
```

Comment:
Would like to see a property named `$ref` tested as well (new test or added to existing one, i'm not picky!)

Reply:
Done